PeriodicCmd command
---
time start_time